### PR TITLE
Bugfix/#2690 fix error message in ua/ru restore password

### PR DESCRIPTION
**Pull request by AlekOK**
The error message is shown in the English language when a site with Ua/Ru language.
I added translate pipe for two values. If we sent a request earlier than one day - we  will receive massage about it.
And if we sent a reques to not exist  email - we also  will receive massage about it.

### CODE REVIEW

File: src/app/main/component/auth/components/restore-password/restore-password.component.html [hunk 1]
```diff
@@ -20,8 +20,11 @@ <h2>{{ 'user.auth.forgot-password.fill-form' | translate }}</h2>
     <div class="validation-email-error" *ngIf="emailField.invalid && emailField.touched">
       <app-error [formElement]="emailField" [emailFieldValue]="emailFieldValue" [controlName]="'email'"></app-error>
     </div>
-    <div class="validation-email-error" *ngIf="emailErrorMessageBackEnd != null">
-      {{ emailErrorMessageBackEnd }}
+    <div class="validation-email-error" *ngIf="emailErrorMessageBackEnd === 'already-sent'">
+      {{ 'user.auth.forgot-password.already-sent' | translate }} : {{ emailFieldValue }}
+    </div>
+    <div class="validation-email-error" *ngIf="emailErrorMessageBackEnd === 'email-not-exist'">
+      {{ 'user.auth.forgot-password.email-not-exist' | translate }} : {{ emailFieldValue }}
```
Comment (JimHolman1199) [inline src/app/main/component/auth/components/restore-password/restore-password.component.html:27]: check for existing error validation component 
if there is no - create one 
so we will not duplicate the code

**Review by JimHolman1199 — CHANGES_REQUESTED** [review]

File: src/app/main/component/auth/components/restore-password/restore-password.component.html (hunk 1 above)
Comment (AlekOK) [reply src/app/main/component/auth/components/restore-password/restore-password.component.html:27]: I created a new component.

File: src/app/main/component/auth/components/message-back-end/message-back-end.component.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  getMessage() {
+    if (this.controlMessage === 'already-sent') {
+      return 'user.auth.forgot-password.already-sent';
+    } else if (this.controlMessage === 'email-not-exist') {
+      return 'user.auth.forgot-password.email-not-exist';
+    } else {
+      return 'attention.error.default';
+    }
+  }
```
Comment (Edw1x) [inline src/app/main/component/auth/components/message-back-end/message-back-end.component.ts:24]: You can change it to:
if;
if;
default return;

But it's optional, as you want.

Comment (AlekOK) [reply src/app/main/component/auth/components/message-back-end/message-back-end.component.ts:24]: Thanks. If I change anything again, I will think about it.

**Comment by AlekOK** [comment]
I create a new branch

**Pull request closed without merging**